scripts/bootstrap/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "bootstrap",
+  "version": "1.0.0",
+  "description": "",
+  "main": "bootstrap.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Grantland Chew",
+  "license": "Platform License",
+  "dependencies": {
+    "parse": "^1.6.13",
+    "promise": "^7.1.1"
+  }
+}
```

Comment:
The syntax is 
```js
  "engines": {
    "node": ">=XXX"
  }
```
Since you do not use es6 feature, probably just check the libraries you use and choose the min version.